lib/garage_client/rails/version.rb

```diff
@@ -1,5 +1,5 @@
 module GarageClient
   module Rails
-    VERSION = "0.0.2"
+    VERSION = "0.0.3"
```

Comment:
Due to this change, applications using Rails 4.0 will not be able to use this gem.
Could you change the gem version to 1.0.0 and change the gemspec Rails dependency version to 5.0 or higher?

Reply:
- Changed rails version dependency https://github.com/tokubai/garage_client-rails/pull/1/commits/7d6ba01d8ac32fd20d1caf10bae1a5f9a90833c1
- Bump version to 1.0.0 https://github.com/tokubai/garage_client-rails/pull/1/commits/5b0b509d973b55dc559d4071b9ff64d6688dd6b2